samples/kotlin/src/main/java/com/forgerock/kotlinapp/FRSessionActivity.kt

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2022 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+package com.forgerock.kotlinapp
+
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.browser.customtabs.CustomTabsIntent
+import androidx.core.content.ContextCompat
+import net.openid.appauth.AuthorizationRequest
+import org.forgerock.android.auth.*
+import java.lang.Exception
+
+
+class FRSessionActivity: AppCompatActivity(), NodeListener<FRSession>, ActivityListener {
+
+    private val status: TextView by lazy { findViewById(R.id.status) }
+    private val loginButton: Button by lazy { findViewById(R.id.login) }
+    private val logoutButton: Button by lazy { findViewById(R.id.logout) }
+    private val classNameTag = FRSessionActivity::class.java.name
+    private var userInfoFragment: UserInfoFragment? = null
+    private var nodeDialog: NodeDialogFragment? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContentView(R.layout.activity_main)
+        updateStatus(showLogin = true)
+        loginButton.setOnClickListener {
+            val journeyName = "Login"
```

Comment:
Could you add some in line comments explaining the functionality of the FRSession.authenticate() call.

I would also suggest pointing to another tree, to make the distinction of calling separate journeys than the ones in the default configuration.